x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings/latest_findings_table.tsx

```diff
@@ -42,62 +39,11 @@ const FindingsTableComponent = ({
 }: FindingsTableProps) => {
   const [selectedFinding, setSelectedFinding] = useState<CspFinding>();
 
-  const columns: Array<
-    EuiTableFieldDataColumnType<CspFinding> | EuiTableActionsColumnType<CspFinding>
-  > = useMemo(
-    () => [
-      {
-        width: '40px',
-        actions: [
-          {
-            name: 'Expand',
-            description: 'Expand',
-            type: 'icon',
-            icon: 'expand',
-            onClick: (item) => setSelectedFinding(item),
-          },
-        ],
-      },
-      {
-        field: 'resource_id',
-        name: TEXT.RESOURCE_ID,
-        truncateText: true,
-        width: '15%',
-        sortable: true,
-        render: resourceFilenameRenderer,
-      },
-      {
-        field: 'result.evaluation',
-        name: TEXT.RESULT,
-        width: '100px',
-        sortable: true,
-        render: resultEvaluationRenderer,
-      },
-      {
-        field: 'rule.name',
-        name: TEXT.RULE,
-        sortable: true,
-      },
-      {
-        field: 'cluster_id',
-        name: TEXT.SYSTEM_ID,
-        truncateText: true,
-        sortable: true,
-      },
-      {
-        field: 'rule.section',
-        name: TEXT.CIS_SECTION,
-        sortable: true,
-        truncateText: true,
-      },
-      {
-        field: '@timestamp',
-        name: TEXT.LAST_CHECKED,
-        truncateText: true,
-        sortable: true,
-        render: timestampRenderer,
-      },
-    ],
+  const columns: [
+    EuiTableActionsColumnType<CspFinding>,
+    ...Array<EuiBasicTableColumn<CspFinding>>
+  ] = useMemo(
+    () => [getExpandColumn<CspFinding>({ onClick: setSelectedFinding }), ...getFindingsColumns()],
```

Comment:
changed because columns are partially shared (for now) between `group-by-none` and `resource-findings`